### ISSUE TITLE
AX_COLOR_01: Contrast issue on gradient backgrounds

### ISSUE THREAD

**Issue by craigfrancis** [GoogleChrome/accessibility-developer-tools#79]
Running the audit on: https://portal.contact-associates.co.uk/

It fails for a colour contrast issue... white text on a gradient background.

Looking at the "Home" link as an example (main nav), the inspector shows:

```
Contrast ratio: 1.00 
AA level (4.54): #767676/#ffffff 
AAA level (7.00): #595959/#ffffff 
```

Not sure where its getting the "1.00", or what the 2 grey colours (#767676 or #595959) are for.

But it should be comparing white against something along the gradient range of #9E4594 (5.6) or #761786 (9.4) - personally I would compare against all colours and report on the worst contrast, but I realise that can get difficult the more colours you have on the gradient.

For reference I am now setting the background directly on the `<a>` to test, as before it was just on the parent `<ul>`... and just setting a static background colour (which is still there for the older browsers) does resolve this issue.

**Comment by Malvoz** [comment]
I don't know what's causing my issue, which seems related, but may it be that the contrast analyzer isn't taking the CSS property/value pair: `brightness: filter(<number-percentage>)` into consideration?

While setting a `<video>` (in which the background doesn't change much behind the stacked element) to `brightness(55%)`, with a IMO very noticable color contrast to an element stacked on top, the analyzer is warning me about low color-contrast, however the color-range which it suggests results in the element being barely visible:<br>
![contrast_issue](https://user-images.githubusercontent.com/26493779/52094811-dc363780-25c0-11e9-8e52-72fd9faf2a76.png)
